plugins/node/opentelemetry-instrumentation-koa/README.md

```diff
@@ -54,6 +54,7 @@ Note that generator-based middleware are deprecated and won't be instrumented.
 | Options            | Type                                | Example              | Description                                                                                              |
 | ------------------ | ----------------------------------- | -------------------- | -------------------------------------------------------------------------------------------------------- |
 | `ignoreLayersType` | `KoaLayerType[]`                    | `['middleware']`     | Ignore layers of specified type.                                                                         |
+| `ignoreLayersName` | `string[]`                          | `['logger']`         | Ignore layers with specified names.                                                                      |
```

Comment:
@MrFabio What would you think about changing this from `ignoreLayersName` to `ignoreLayers` and copying the behaviour of the `ignoreLayers` config from instrumentation-express, given that instrumentation-express already has a very similar config option with (mostly) the same behaviour?